jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/sam/SamCommon.kt

```diff
@@ -32,7 +32,7 @@ class SamCommon {
         const val SAM_INVALID_OPTION_SUBSTRING = "no such option"
 
         // Inclusive
-        val expectedSamMinVersion = SemVer("0.14.1", 0, 14, 1)
```

Comment:
This will need a changelog entry of just like we did when we raised from 0.7,

I'd make a dedicated commit for this change

Reply:
The whole nodejs feature branch will be merged into master in one commit. Is it worth having a separate commit for the min version bump?